<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Administrate</class>
 <widget class="QWidget" name="Administrate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="pageName_1">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>-10</y>
     <width>341</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: orange;
font-size: 35px;
font-family: Roboto;</string>
   </property>
   <property name="text">
    <string>Administrate trains</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>830</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>← Go back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addTrainButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchField">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>150</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="optionsButton">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>150</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="trainTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>219</y>
     <width>1460</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #616161, stop: 0.5 #505050,
                                      stop: 0.6 #434343, stop:1 #656565);
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
}

QTableView {
    selection-background-color: #dedede;

	selection-color: dark-grey;

	font-size: 25px;
}


/* style the sort indicator */
QHeaderView::down-arrow {
    image: url(down_arrow.png);
}

QHeaderView::up-arrow {
    image: url(up_arrow.png);
}

QTableView::item {
}
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearSearchButton">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>150</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="textFileButton">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>850</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
    spacing: 5px;
	font-size: 15px;
}

QRadioButton::indicator {
    width: 20px;
    height: 20px;
	margin-right: 5px;
}</string>
   </property>
   <property name="text">
    <string>text and binary file</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="arrayButton">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>850</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
    spacing: 5px;
	font-size: 15px;
}

QRadioButton::indicator {
    width: 20px;
    height: 20px;
	margin-right: 5px;
}</string>
   </property>
   <property name="text">
    <string>array</string>
   </property>
  </widget>
  <widget class="QPushButton" name="trainEditButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>✎</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeTrainButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ff4f4f;
border-style: outset;
    border-color: beige;
}

QPushButton:disabled {
background-color:#ff7d7d;
}</string>
   </property>
   <property name="text">
    <string>Х</string>
   </property>
  </widget>
  <widget class="QLabel" name="intervalLabel">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>120</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #4c9e33;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
